### PR TITLE
Divider

### PR DESCRIPTION
**Pull request by SebastianStehle**
Implements mj-divider.

Also implement UnitParser, which provides the unit as enum to be allocation free. We should use the parser whenever we need a number.

### CODE REVIEW

File: Mjml.Net/Components/Body/DividerComponent.cs [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                case "left":
+                    return "0px";
+                case "right":
+                    return "0px 0px 0px auto";
+                default:
+                    return "0px auto";
+            }
+        }
+
+        private static string GetOutlookWidth(IHtmlRenderer renderer, INode node)
```
Comment (LiamRiddell) [inline Mjml.Net/Components/Body/DividerComponent.cs:95]: From memory, this function appears in multiple components. I think we should move this into a NodeHelper possibly.

**Review by LiamRiddell — CHANGES_REQUESTED** [review]
Majority looks good but maybe worth doing a quick search on MJML source code regarding the GetOutlookWidth as I think we could lift that up to a NodeExtension helper.

**Comment by LiamRiddell** [comment]
I like the UnitParser implementation with the allocation free Parsing.

**Comment by SebastianStehle** [comment]
I have not found int anywhere else.

**Pull request merged**